browser/src/control/Control.UIManager.ts

```diff
@@ -55,68 +72,91 @@ L.Control.UIManager = L.Control.extend({
-			if (event.target.parentElement.id === 'toolbar-up') // checks if clicked on empty part of the toolbar on tabbed view
-				that.map.fire('editorgotfocus');
+		$('#toolbar-wrapper').on('click', (event) => {
+			const target = event.target as HTMLElement;
+			if (target.parentElement?.id === 'toolbar-up') // checks if clicked on empty part of the toolbar on tabbed view
+				this.map.fire('editorgotfocus');
 		});
 
-		$('.main-nav').on('click', function (event) {
-			if (event.target.parentElement.nodeName === 'NAV' || // checks if clicked on an container of an element
-				event.target.nodeName === 'NAV' || // checks if clicked on navigation bar itself
-				event.target.parentElement.id === 'document-titlebar') { // checks if clicked on the document titlebar container
-				that.map.fire('editorgotfocus');}
+		$('.main-nav').on('click', (event) => {
+			const target = event.target as HTMLElement;
+			if (target.parentElement?.nodeName === 'NAV' || // checks if clicked on an container of an element
+				target.nodeName === 'NAV' || // checks if clicked on navigation bar itself
+				target.parentElement?.id === 'document-titlebar') { // checks if clicked on the document titlebar container
+				this.map.fire('editorgotfocus');}
 		});
 
 		this.map.on('updateviewslist', this.onUpdateViews, this);
 
 		this.map['stateChangeHandler'].setItemValue('toggledarktheme', 'false');
 		this.map['stateChangeHandler'].setItemValue('invertbackground', 'false');
 		this.map['stateChangeHandler'].setItemValue('showannotations', 'true');
-	},
+	}
 
 	// UI initialization
 
-	getCurrentMode: function() {
+	/**
+	 * Returns the current UI mode ("notebookbar" or "classic").
+	 */
+	getCurrentMode(): string {
 		// no notebookbar on mobile
 		if (window.mode.isMobile())
 			return 'classic';
 
 		return this.shouldUseNotebookbarMode() ? 'notebookbar' : 'classic';
-	},
+	}
 
-	getHighlightMode: function() {
+	/**
+	 * Returns if the highlight of column/row is enabled.
+	 */
+	getHighlightMode(): boolean {
 		return window.prefs.getBoolean('ColumnRowHighlightEnabled', false);
-	},
+	}
 
-	setHighlightMode: function( newState ) {
+	/**
+	 * Changes status of the highlight of column/row is enabled.
+	 */
+	setHighlightMode(newState: boolean): void {
 		window.prefs.set('ColumnRowHighlightEnabled', newState);
-		let highlightState = newState? 'true' : 'false';
+		const highlightState = newState? 'true' : 'false';
 		this.map['stateChangeHandler'].setItemValue('columnrowhighlight', highlightState);
 		this._map.fire('commandstatechanged', {commandName : 'columnrowhighlight', state : highlightState});
-	},
+	}
 
-	shouldUseNotebookbarMode: function() {
-		let forceCompact = window.prefs.getBoolean('compactMode', null);
+	/**
+	 * Determines whether to use notebookbar mode based on user preferences.
+	 */
+	shouldUseNotebookbarMode(): boolean {
+		const forceCompact = window.prefs.get('compactMode', null); // getBoolean() does not accept null as the default value
```

Comment:
Was getBoolean before, I don't see any modification of setter, also comment is about old version.
Is that intentional?

Reply:
window.prefs.set('compactMode', uiMode.mode === 'classic'); is where it is set - which stores that as string

Reply:
Maybe I should have worded the comment differently, like "can't use getBoolean() here, because that wouldn't accept null as the default value"